From f66206679a0ad604f13673559f230160cd3d1189 Mon Sep 17 00:00:00 2001
From: Max Filippov <jcmvbkbc@gmail.com>
Date: Fri, 14 Aug 2015 02:45:02 +0300
Subject: [PATCH 2/3] xtensa: use unwind-dw2-fde-dip instead of unwind-dw2-fde

This allows having exception cleanup code in binaries that don't
register their unwind tables.

2015-08-18  Max Filippov  <jcmvbkbc@gmail.com>
libgcc/
	* config/xtensa/t-xtensa (LIB2ADDEH): Replace unwind-dw2-fde
	with unwind-dw2-fde-dip.

Signed-off-by: Max Filippov <jcmvbkbc@gmail.com>
---
Backported from: r226963

 libgcc/config/xtensa/t-xtensa |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libgcc/config/xtensa/t-xtensa
+++ b/libgcc/config/xtensa/t-xtensa
@@ -13,4 +13,4 @@
 LIB2ADD = $(srcdir)/config/xtensa/lib2funcs.S
 
 LIB2ADDEH = $(srcdir)/config/xtensa/unwind-dw2-xtensa.c \
-   $(srcdir)/unwind-dw2-fde.c $(srcdir)/unwind-sjlj.c $(srcdir)/unwind-c.c
+   $(srcdir)/unwind-dw2-fde-dip.c $(srcdir)/unwind-sjlj.c $(srcdir)/unwind-c.c
